packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -538,6 +562,19 @@ export default class Health extends Component {
 
   _getSrUrl = sr => `srs/${sr.id}`
 
+  _getDuplicatedMacAddresses = createSelector(
+    () => this.props.vifsByMac,
+    vifsByMac => {
+      const duplicatedMacAddresses = []
+      for (const [macAddress, vifs] of Object.entries(vifsByMac)) {
+        if (vifs.length > 1) {
+          duplicatedMacAddresses.push(macAddress)
+        }
+      }
+      return duplicatedMacAddresses
+    }
```

Comment:
Wrap it with `createCollectionWrapper` and sort the MAC addresses so that this selector always return the same reference when the collection doesn't change.